tests/bluetooth/tester/default.conf

```diff
@@ -9,5 +9,17 @@ CONFIG_BT_ATT_PREPARE_COUNT=4
 CONFIG_BT_GATT_CLIENT=y
 CONFIG_BT_L2CAP_DYNAMIC_CHANNEL=y
 CONFIG_BT_PRIVACY=y
+
+CONFIG_BT_MESH=y
+CONFIG_BT_MESH_PROV=y
+CONFIG_BT_MESH_RELAY=y
+CONFIG_BT_MESH_PB_ADV=y
+CONFIG_BT_MESH_PB_GATT=y
+CONFIG_BT_MESH_GATT_PROXY=y
+CONFIG_BT_MESH_SUBNET_COUNT=5
+CONFIG_BT_MESH_APP_KEY_COUNT=5
+CONFIG_BT_MESH_MODEL_GROUP_COUNT=5
```

Comment:
Same with this. I think most of these you can just leave at their default values, i.e. not mention them at all in the config.

Reply:
ok, I'll use default values then